mmv1/products/compute/NetworkEndpoints.yaml

```diff
@@ -0,0 +1,139 @@
+description: |
+  A set of network endpoints belonging to a network endpoint group (NEG). A
+  single network endpoint represents a IP address and port combination that is
+  part of a specific (NEG). NEGs are zonal collections of these endpoints for
+  GCP resources within a single subnet. **NOTE**: Network endpoints cannot be
+  created outside of a network endpoint group.
+
+  This resource is authoritative for a single NEG. Any endpoints not specified
+  by this resource will be deleted when the resource configuration is applied.
+immutable: true
+create_verb: :POST
+create_url: projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}/attachNetworkEndpoints
+delete_verb: :POST
+delete_url: projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}/detachNetworkEndpoints
+read_verb: :POST
+self_link: projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}/listNetworkEndpoints
+identity:
+  - networkEndpointGroup
+collection_url_key: 'items'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/load-balancing/docs/negs/'
+  api: 'https://cloud.google.com/compute/docs/reference/rest/beta/networkEndpointGroups'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    kind: 'compute#operation'
+    path: 'name'
+    base_url: 'projects/{{project}}/zones/{{zone}}/operations/{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'targetLink'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'status'
+    complete: 'DONE'
+    allowed:
+      - 'PENDING'
+      - 'RUNNING'
+      - 'DONE'
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error/errors'
+    message: 'message'
+id_format: '{{project}}/{{zone}}/{{network_endpoint_group}}/endpoints'
+import_format:
+  - 'projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}'
+mutex: networkEndpoint/{{project}}/{{zone}}/{{network_endpoint_group}}
+exclude_validator: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_endpoints'
+    primary_resource_id:
+      'default-endpoints'
+      # Fine-grained resource need different autogenerated tests, as
+      # we need to check destroy during a test step where the parent resource
+      # still exists, rather than during CheckDestroy (when read returns
+      # nothing because the parent resource has then also been destroyed)
+    skip_test: true
+    vars:
+      neg_name: 'my-lb-neg'
+      instance_name: 'endpoint-instance'
+      network_name: 'neg-network'
+      subnetwork_name: 'neg-subnetwork'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  pre_delete: templates/terraform/pre_delete/compute_network_endpoints.go.erb
+  # custom_import: templates/terraform/custom_import/compute_network_endpoints.go.erb
```

Comment:
```suggestion
```

Reply:
Oops, thought I had already deleted that...